AzureFunctionForSplunk/.gitignore

```diff
@@ -2,7 +2,7 @@
 ## files generated by popular Visual Studio add-ons.
 
 # Azure Functions localsettings file
-local.settings.json
+*local.settings.json
```

Comment:
My local settings has secrets in it - a security risk. Is it necessary to check it in?

Reply:
The * doesn't comment it out so it is still in effect, but I don't see any reason to write it that way nor do I remember why I did that. Let's change it back